statistics/handle/bootstrap.go

```diff
@@ -362,6 +456,36 @@ func (h *Handle) initStatsBuckets(cache *statsCache) error {
 	return nil
 }
 
+func (h *Handle) InitStatsLite(is infoschema.InfoSchema) (err error) {
```

Comment:
Please add some comments to explain the differences between `InitStatsLite` and `InitStats`.

Reply:
Or can we merge these 2 code paths into one, e.g. `IndexStats(..., lite bool)`, `initStatsHistograms(..., light bool)`

Reply:
Updated.